src/minimal_unified_api.py

```
@@ -0,0 +1,177 @@
+            'joy', 'love', 'nervousness', 'optimism', 'pride', 'realization',
+            'relief', 'remorse', 'sadness', 'surprise', 'neutral'
+        ]
+        emotion_scores = predictions[0].tolist()
+
+        result = {
+            "text": text,
+            "emotions": dict(zip(emotion_labels, emotion_scores)),
+            "predicted_emotion": emotion_labels[emotion_scores.index(max(emotion_scores))]
+        }
+
+        return result
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)